### PR TITLE
Feature/collapse location cards

### PR DESCRIPTION
**Pull request by schamp**
Update based on designs: https://www.figma.com/file/JGqemgZwlG9e5DQbZpdKgy/Redesign?node-id=2269%3A88

After:
Desktop:
image
375x800: (note search bar is full width)
image
320x568: (note fixed search bar -- doesn't extend beyond screen)
image
320x568 (scrolled):
image

### CODE REVIEW

**Comment by schamp** [comment]
[Collapse location cards on homepage](https://trello.com/c/10mKstRR/1234-collapse-location-cards-on-homepage)

**Comment by schamp** [comment]
If you have any guidance about a better way to organize the styles or apply properties to achieve the desired result, let me have it!  I got this working, but my CSS game isn't as strong as I'd like and there may well be better ways to accomplish this.

**Comment by joshziman** [comment]
Looking good @schamp! I wasn't able to load the cards, perhaps for the reason you mentioned in Trello. Judging from the screenshots, on mobile, the only thing I'd request changed is to have the card "full bleed" to the edge of the screen, i.e. there wouldn't be a gap right around "VA" in the screenshot.

**Comment by schamp** [comment]
Cool, thanks, I'll see what I can do about that.

File: src/screens/HomePage/HomePage.style.tsx [hunk 1]
```diff
@@ -35,30 +29,19 @@ export const Section = styled.div`
...
-    font-size: 1.125rem;
-  }
-
-  ${StyledLink},${SkeletonWrapper} {
-    &:last-of-type {
-      margin: .5rem 0 2.5rem;
-    }
-  }
+  flex-direction: row;
+  gap: 0.75rem;
```
Comment (mimecuvalo) [inline src/screens/HomePage/HomePage.style.tsx:33]: TIL about `gap` - neat!

File: src/components/RegionItem/RegionItem.style.tsx [hunk 2]
```diff
@@ -51,9 +65,13 @@ export const SharedWrapperStyles = css`
...
   align-items: center;
   border: 1px solid ${COLORS.LIGHTGRAY};
-  width: 330px;
-  padding: 1.25rem 1.75rem 1.25rem 1.25rem;
+  max-width: 296px;
+  padding: 1rem 1.25rem 1rem 1.25rem;
   border-radius: 4px;
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    width: 296px;
```
Comment (mimecuvalo) [inline src/components/RegionItem/RegionItem.style.tsx:73]: nit: might be good to make this a constant with above

**Review by mimecuvalo — APPROVED** [review]
LGTM with caveat that i'm still ramping on this code
(and also, hopefully we'll have constants long-term in our other codebases vs. hard-coded numbers but I'm not worried for the CAN site in this regard)

File: src/assets/theme/searchbar.ts [hunk 3]
```diff
@@ -16,7 +16,7 @@ export interface SearchbarTheme {
 
 export const searchbar: SearchbarTheme = {
   mobileClosedWidth: `${mobileWidth}px`,
-  mobileClosedPadding: '0.75rem 0.5rem 1.5rem',
+  mobileClosedPadding: '0.75rem 0.5rem 1rem',
```
Comment (chasulin) [inline src/assets/theme/searchbar.ts:19]: lets keep this `1.5rem` to keep it close to [the mocks](https://www.figma.com/file/JGqemgZwlG9e5DQbZpdKgy/Redesign?node-id=1476%3A143)

File: src/components/Search/SearchAutocomplete/SearchAutocomplete.style.tsx [hunk 4]
```diff
@@ -55,12 +55,14 @@ const MobileWrapperClosed = css`
...
   padding: ${props => props.theme.searchbar.mobileClosedPadding};
-  margin-left: ${props => props.theme.searchbar.mobileClosedMarginLeft};
   max-width: ${props => props.theme.searchbar.mobileClosedMaxWidth};
+  margin: auto;
 `;
 
 export const Wrapper = styled.div<{ $isOpen: boolean }>`
   ${({ $isOpen }) => ($isOpen ? MobileWrapperOpened : MobileWrapperClosed)};
+  width: 100%;
+  max-width: ${mobileWidth}px;
```
Comment (chasulin) [inline src/components/Search/SearchAutocomplete/SearchAutocomplete.style.tsx:65]: this^ results in the menu not being full bleed on mobile:

Screen Shot 2021-05-17 at 6 39 36 PM

File: src/screens/HomePage/HomePage.style.tsx (hunk 1 above)
Comment (chasulin) [reply src/screens/HomePage/HomePage.style.tsx:33]: ooooo same!!

File: src/components/Search/SearchAutocomplete/SearchAutocomplete.style.tsx [hunk 5]
```diff
@@ -55,12 +55,14 @@ const MobileWrapperClosed = css`
   height: unset;
   width: ${props => props.theme.searchbar.mobileClosedWidth};
   padding: ${props => props.theme.searchbar.mobileClosedPadding};
-  margin-left: ${props => props.theme.searchbar.mobileClosedMarginLeft};
   max-width: ${props => props.theme.searchbar.mobileClosedMaxWidth};
+  margin: auto;
```
Comment (chasulin) [inline src/components/Search/SearchAutocomplete/SearchAutocomplete.style.tsx:59]: oo any reason to edit these margins? nixing the margin-left and adding margin:auto throws off the spacing and width of the searchbar slightly

prod (search bar smaller and set to the right)-
Screen Shot 2021-05-17 at 6 49 11 PM

feature branch-
Screen Shot 2021-05-17 at 6 49 20 PM

File: src/components/RegionItem/RegionItem.style.tsx (hunk 2 above)
Comment (schamp) [reply src/components/RegionItem/RegionItem.style.tsx:73]: good idea, thanks

File: src/components/Search/SearchAutocomplete/SearchAutocomplete.style.tsx (hunk 4 above)
Comment (schamp) [reply src/components/Search/SearchAutocomplete/SearchAutocomplete.style.tsx:65]: nice catch, thanks.  removing both is fine, I didn't need them after all, so long as I change the 'closed' one in the theme from '350px' to '100%';

File: src/components/Search/SearchAutocomplete/SearchAutocomplete.style.tsx (hunk 5 above)
Comment (schamp) [reply src/components/Search/SearchAutocomplete/SearchAutocomplete.style.tsx:59]: reverting this seemed to work, thanks for the callout.

**Comment by schamp** [comment]
fixed the padding to match the design more closely (was too small)
image

fixed full bleed on mobile search open: 
image

looks good small too:
image

search on 320px screen:
image

search on 375px screen (back to original on prod):
image

region cards now full bleed on mobile:  
image

I think that's everything, @chasulin do you mind confirming and stamping when you're happy with it?  then I'll merge.

**Review by pnavarrc — APPROVED** [review]
Looks good, 🚢 !

**Pull request merged**